recipes/pudb/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2016.2" %}
+
+package:
+  name: pudb
+  version: {{ version }}
+
+source:
+  fn: pudb-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/50/1a/d9b692e32afff09ccb5aa33c3d51c6d5a80efbb59de90307b33601e7fa62/pudb-{{ version }}.tar.gz
```

Comment:
We should switch to pypi.io. The problem with these URLs is the hashes change every version. So, it defeats the jinja templating.